CMakeLists.txt

```diff
@@ -280,6 +281,11 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "(Apple)?Clang")
 	endif()
 endif()
 
+if(BUILD_BENCHMARKS)
+	set(CATCH2_LIBRARY catch2)
```

Comment:
no need to set this since the library name is always `catch2` :)

Reply:
I was following the pattern above:
https://github.com/minetest/minetest/blob/47cf257c4098f087d4dc46ac28ddb39141222b0c/CMakeLists.txt#L267

There is some benefit to having CATCH2_LIBRARY as a grep-able option with a single source of truth (in case someone wants to swap it out, for example), but it's not that big a deal.

Reply:
Fixed